packages/smooth_app/lib/pages/user_management/login_page.dart

```diff
@@ -91,151 +89,163 @@ class _LoginPageState extends State<LoginPage> {
           alignment: Alignment.topCenter,
           child: SizedBox(
             width: size.width * 0.7,
-            child: Column(
-              crossAxisAlignment: CrossAxisAlignment.center,
-              mainAxisAlignment: MainAxisAlignment.center,
-              children: <Widget>[
-                const Spacer(flex: 4),
-
-                Text(
-                  appLocalizations.sign_in_text,
-                  textAlign: TextAlign.center,
-                  style: theme.textTheme.headline1?.copyWith(
-                    fontSize: 20.0,
-                    fontWeight: FontWeight.w700,
-                    color: theme.colorScheme.onSurface,
+            child: AutofillGroup(
```

Comment:
This file looks like a lot of changes but I only wrapped the `Column `inside a `AutofillGroup`